estuary_updater/handlers/koji.py

```diff
@@ -76,6 +78,29 @@ def build_tag_handler(self, msg):
         # Check to see if we want to process this tag
         if not build:
             return
+
+        if build.__label__ == ModuleKojiBuild.__label__:
+            extra_json = json.loads(msg['body']['msg']['info']['extra'])
+            module_extra_info = extra_json.get('typeinfo', {}).get('module')
+            module_build_tag_name = module_extra_info.get('content_koji_tag')
+            if module_build_tag_name:
+                try:
+                    tag_info = self.koji_session.getTag(module_build_tag_name, strict=True)
+                except Exception:
+                    log.error('Failed to get tag {0}'.format(module_build_tag_name))
+                    raise
+                module_build_tag = KojiTag.create_or_update(dict(
+                    id_=tag_info[0]['tag_id'],
+                    name=module_build_tag_name
+                ))[0]
+                module_build_tag.module_builds.connect(build)
+
+                for item in tag_info:
+                    module_component = KojiBuild.get_or_create(dict(
+                        id_=item['build_id']
+                    ))[0]
+                    build.components.connect(module_component)
```

Comment:
For each build associated to that tag. You can call `get_or_create_build` from base.py to update the Neo4j build objects. That would take care of the recursive part in the scrapers.